expression/builtin.go

```diff
@@ -582,6 +584,14 @@ type functionClass interface {
 	verifyArgsByCount(l int) error
 }
 
+// operatorClass is the interface for a operator extended from functionClass
+type operatorClass interface {
```

Comment:
I think class should decide the display name on their own. Maybe

```
... interface {
  DisplayName() string
}
```

Or even simpler:

```
_, ok := funClass.(fmt.Stringer)
```

Just a suggestion, I am fine with the current solution.

Reply:
Currently only the operators need to be displayed more visually, but I've come up with a more customizable solution:
```go
type functionClassWithName interface {
    getDisplayName() string
}
```
then in `GetDisplayName`:
```go
switch c := funClass.(type) {
    case functionClassWithName:
        return c.getDisplayName()
    case operatorClass:
        return c.getOpcode().FormatToString()
    default:
        return name
}
```
So when some class want to decide the display name, they can define `getDisplayName`, and others just ignore it or define a `getOperator`.

Reply:
Currently only the operators need to be displayed more visually, but I've come up with a more customizable solution:
```go
type functionClassWithName interface {
    getDisplayName() string
}
```
then in GetDisplayName:
```go
switch c := funClass.(type) {
    case functionClassWithName:
        return c.getDisplayName()
    case operatorClass:
        return c.getOpcode().FormatToString()
    default:
        return name
}
```
So when some class want to decide the display name, they can define getDisplayName, and others just ignore it or define a getOperator.

Reply:
That is what I mean, except operator is not special, IMO. There is a fallback, and operator function could define its own `getDisplayName()`, i.e. function returning `=` or `+` in this case.

Reply:
I think for operators, `getOperator` is more general and we could reuse the operator information (including display name) from the parser, otherwise we should introduce some duplicate strings.

Reply:
Exposing is good, but I don't known where it can be used. I am not that familiar with expression.

Reply:
Sorry, I got your idea now. Done via 7905f1b0cb156f5c6074a79bc7287db88bfac3b2.